CHANGES.txt

```diff
@@ -1,6 +1,8 @@
 0.4 (unreleased)
 ----------------
 
+- Add new command ``pyramid-chameleon-precompile``.
+
```

Comment:
Strictly it is not my change, but I thought it was necessary, so I added it 😄

Reply:
Thanks!